### PR TITLE
Nikolai/tooltip-for-withdrawal

### PR DESCRIPTION
**Pull request by nikolai-deriv**

### CODE REVIEW

**Review by kseniya-deriv — DISMISSED** [review]

File: src/javascript/app/pages/user/account/statement/statement.ui.js [hunk 1]
```diff
@@ -63,7 +63,12 @@ const StatementUI = (() => {
         ], columns, 'data');
         $statement_row.children('.credit').addClass(credit_debit_type);
         $statement_row.children('.date').addClass('pre');
-        $statement_row.children('.desc').html(`${statement_data.desc}<br>`);
+        $statement_row.children('.desc').html(`<span>${statement_data.desc}</span><br>`);
+
+        // add processing time popup for withdrawal
+        if (transaction.action_type === 'withdrawal') {
+            $statement_row.children('.desc').find('span').attr('data-balloon', transaction.withdrawal_details);
```
Comment (carol-binary) [inline src/javascript/app/pages/user/account/statement/statement.ui.js:70]: What is `data-balloon`?

Comment (carol-binary) [reply src/javascript/app/pages/user/account/statement/statement.ui.js:70]: Do we use `data-balloon` anywhere else in `binary-static`?

Comment (nikolai-deriv) [reply src/javascript/app/pages/user/account/statement/statement.ui.js:70]: Its an attribute that adds a popup to an element

Comment (carol-binary) [reply src/javascript/app/pages/user/account/statement/statement.ui.js:70]: Found it :)

**Review by carol-binary — APPROVED** [review]

**Review by nijil-binary — DISMISSED** [review]

**Review by Daniil-Deriv — APPROVED** [review]

**Review by carol-binary — DISMISSED** [review]

**Review by nijil-binary — APPROVED** [review]

**Pull request closed without merging**